modules/extension/xsd/xsd-core/src/main/java/org/geotools/xsd/Parser.java

```diff
@@ -65,12 +66,22 @@
 
     private static final String SAX_PROPERTY_PREFIX = "http://xml.org/sax/properties/";
 
+    private static final String JAXP_PROPERTY_PREFIX = "http://www.oracle.com/xml/jaxp/properties/";
```

Comment:
Would it make sense to have a default setup that won't make the system go OOM easily?

Reply:
Indeed, adding 100 as default value, it is a good default?

Reply:
Seems reasonable yes.